src/columnresizing.js

```diff
@@ -147,7 +147,8 @@ function domCellAround(target) {
 }
 
 function edgeCell(view, event, side) {
-  let {pos} = view.posAtCoords({left: event.clientX, top: event.clientY})
+  const buffer = side == 'right' ? -5 : 5; // Fixes finicky bug where posAtCoords could return wrong pos.
```

Comment:
Thanks for the fix! Can we use `handleWidth` instead of `5`? https://github.com/ProseMirror/prosemirror-tables/blob/master/src/columnresizing.js#L10